### PR TITLE
Show no more than one FIFO error per session.

### PR DESCRIPTION
**Pull request by skidau**

### CODE REVIEW

**Comment by mimimi085181** [comment]
Hmm, is it really a good idea to only show the 1st error? Wouldn't it be better to show each error once? So if a title has several errors, and a tester runs it, all errors can be reported instead of just the 1st one.

Also, how about changing the error texts, so it becomes clear that there won't be further notice about the error? If the error happens all the time, but is only displayed at the start, for inexperienced users, it would be unexpected that issues much later into the game are related to it.

**Comment by skidau** [comment]
There is only one error.  When there is a string of panic alerts, it means the FIFO is reading garbage data, not FIFO data.  Usually what occurs shortly after is that the game hangs as the error text predicts (FIFO issue).  Although it shows only the first error, the rest of the errors are logged.  This PR is really to prevent the soft lockup where panic alerts take over the game and the emulator cannot be closed without silencing the error completely.

**Comment by degasus** [comment]
Better: This PR prevents the user from disabling PanicAlerts.

File: Source/Core/VideoCommon/OpcodeDecoding.cpp [hunk 1]
```diff
@@ -283,7 +290,10 @@ u8* OpcodeDecoder_Run(DataReader src, u32* cycles, bool in_display_list)
 			}
 			else
 			{
-				UnknownOpcode(cmd_byte, opcodeStart, is_preprocess);
+				if (!g_bFifoErrorSeen)
+					UnknownOpcode(cmd_byte, opcodeStart, is_preprocess);
+				NOTICE_LOG(VIDEO, "FIFO: Unknown Opcode(0x%02x @ %p, preprocessing = %s)", cmd_byte, opcodeStart, is_preprocess ? "yes" : "no");
```
Comment (degasus) [inline Source/Core/VideoCommon/OpcodeDecoding.cpp]: I think an ERROR_LOG would be better.

**Comment by degasus** [comment]
else, LGTM

**Comment by psennermann** [comment]
This could be a workaround for issue 8042 ?

**Comment by JMC47** [comment]
I think this is a solution for games like Dave Mirra, too. I mean, I'm not happy about it, but this way we know something's wrong without having to turn off panic handlers.

**Comment by phire** [comment]
> This could be a workaround for issue 8042 ?

No, there are plenty of other places in the code where you might enter into an infinite panic handler loop.
A global solution is still needed.

**Pull request merged**